doc/Remove-Account.md

```diff
@@ -11,13 +11,13 @@ with your web browser.
 You will need to be logged in at the time.
 
 You will be asked for your password to confirm the request.
-If this matches your stored password, your account will immediately be blocked to all probing.
-Unlike some social networks we do **not** hold onto it for a grace period in case you change your mind.
-All your content and user data, etc is instantly removed.
-For all intents and purposes, the account is gone in moments.  
-
-We then send out an "unfriend" signal to all of your contacts.
-This signal deletes all content on those networks.
-Unfortunately, due to limitations of the other networks, this only works well with Friendica contacts.
-We allow four days for this, in case some servers were down and the unfriend signal was queued.
-After this, we finish off deleting the account.
+If this matches your stored password, your account will immediately be marked as deleted.
+There is no grace period, this action cannot be reverted.
+Most of your content and user data will be deleted on short term.
+
+We then send out a notification about the account removal to all of your contacts so that they can do the same with their copy of your data.
+
+For technical reasons some of your user data is still needed to transmit this removal message.
+This last data will be deleted after a period of around seven days.
```

Comment:
last => remaining

Reply:
"last" is not completely wrong, isn't it? (But I will change it, since "remaining" sounds better)

Reply:
"Last" is for things you can count. "Data" is innumerable, so you can't use "last". You could have written "This last piece of data" but it's unnecessarily complex.